src/index.js

```diff
@@ -20,13 +36,24 @@ export default {
             throw new Error('Unsupported browser!');
 
         const browserNameString = browserName.match(/([^@:]+)/);
+        const browserFormalName = browserNameString[1].trim();
         let version = browserName.match(/@([^:]+)/);
         let platform = browserName.match(/:(.+)/);
 
         version = version ? version[1] : undefined; // eslint-disable-line no-undefined
         platform = platform ? platform[1] : undefined; // eslint-disable-line no-undefined
-        const browser = await new Builder().forBrowser(browserNameString[1], version, platform).usingServer(this.seleniumServer).build();
 
+        const builder = new Builder().forBrowser(browserFormalName, version, platform).usingServer(this.seleniumServer);
+        const optionSetter = optionSetterMaps[browserFormalName];
+
+        if (optionSetter && existsSync(this.capabilities)) {
+            const caps = JSON.parse(readFileSync(this.capabilities))[browserName.trim()];
```

Comment:
From line 26, the capabilities can be in a file or in `process.env.SELENIUM_CAPABILITIES`. If it is in an environment varialbe will this work? It should also be mentioned in the readme that it is possible to sent it in an env var.
The default for this should be no custom capabilities set.

Reply:
SELENIUM_CAPABILITIES is used to override the default path of capabilities file. By default is capabilities.json. I just provide this env-var in case some one may need to use this (like concurrent job using different configuration)

Reply:
Cool, somehow I wrongly read that it could also be passed as text into the env var.